vega_sim/parameter_test/parameter/configs.py

```diff
@@ -261,15 +261,34 @@
 )
 
 MARKET_PARAMETER_DEMO = SingleParameterExperiment(
```

Comment:
Instead of replacing `MARKET_PARAMETER_DEMO` could we add a new experiment (e.g. `MIN_LP_STAKE_QUANTUM`).